### PR TITLE
feat: re-order the homefeed

### PR DESCRIPTION
**Pull request by egdbear**
This PR updates the section order.

Ticket [DIA-947]

[DIA-947]: https://artsyproduct.atlassian.net/browse/DIA-947?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dblandin — APPROVED** [review]
Technically, this looks fine! As this is the first significant server-driven reordering of home view components targeting current app users, I might get a heads up in [our #pdde-art-advisor Slack channel](https://artsy.slack.com/archives/C07ANEV7RNV).

**Review by anandaroop — APPROVED** [review]
Nice 👍🏽

File: src/schema/v2/homeView/zones/default.ts [hunk 1]
```diff
@@ -31,16 +31,16 @@ const SECTIONS: HomeViewSection[] = [
...
+  HeroUnits,
   ActiveBids,
   AuctionLotsForYou,
   Auctions,
   LatestAuctionResults,
   GalleriesNearYou,
   LatestArticles,
   News,
-  CuratorsPicksEmerging,
   MarketingCollections,
```
Comment (anandaroop) [inline src/schema/v2/homeView/zones/default.ts:44]: **question (non-blocking):** Just noticing the [comment here](https://artsy.slack.com/archives/C07ANEV7RNV/p1729860898364819) and wondering if `MarketingCollections` was meant to be removed here as well.

**Review by anandaroop — APPROVED** [review]
Just one follow-up q

**Pull request merged**